README.md

```diff
@@ -85,49 +85,51 @@ Example of running Snap keystone collector and writing data to file.
 
 Download an [example Snap global config](examples/cfg/cfg.json) file.
 ```
-$ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-keystone/master/examples/cfg/cfg.json
+$ curl -sfLO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-keystone/master/examples/cfg/cfg.json > /etc/snap/snapteld.conf
 ```
 Ensure to provide your Keystone instance address and credentials.
 
 Ensure [Snap daemon is running](https://github.com/intelsdi-x/snap#running-snap) with provided configuration file:
-* command line: `snapd -l 1 -t 0 --config cfg.json&`
+* initd: `sudo service snap-telemetry start`
+* systemd: `sudo systemctl start snap-telemetry`
+* command line: `sudo snapteld -l 1 -t 0 &`
```

Comment:
Now it's just above in line [88](https://github.com/intelsdi-x/snap-plugin-collector-keystone/pull/12/files/76f345e37487ddfabe7d1ff212a8b21dffcb4e34#diff-04c6e90faac2675aa89e2176d2eec7d8R88 ) Maybe we should link to  [this file](https://github.com/IRCody/snap/blob/31347d829dd3ae57da631904a38f1649972670cb/docs/SNAPTELD_CONFIGURATION.md) in order to clarify how global config works.